src/main/scala/sbtghactions/GenerativePlugin.scala

```diff
@@ -713,61 +713,45 @@ ${indent(jobs.map(compileJob(_, sbt)).mkString("\n\n"), 1)}"""
 
     githubWorkflowGenerate := {
       val ciContents = generateCiContents.value
+      val includeClean = githubWorkflowIncludeClean.value
       val cleanContents = readCleanContents.value
 
       val ciYml = ciYmlFile.value
       val cleanYml = cleanYmlFile.value
 
-      val ciWriter = new BufferedWriter(new FileWriter(ciYml))
-      try {
-        ciWriter.write(ciContents)
-        ciWriter.newLine()
-      } finally {
-        ciWriter.close()
-      }
+      IO.write(ciYml, ciContents)
 
-      val cleanWriter = new BufferedWriter(new FileWriter(cleanYml))
-      try {
-        cleanWriter.write(cleanContents)
-        cleanWriter.newLine()
-      } finally {
-        cleanWriter.close()
-      }
+      if(includeClean)
+        IO.write(cleanYml, cleanContents)
     },
 
     githubWorkflowCheck := {
       val expectedCiContents = generateCiContents.value
+      val includeClean = githubWorkflowIncludeClean.value
       val expectedCleanContents = readCleanContents.value
 
       val ciYml = ciYmlFile.value
       val cleanYml = cleanYmlFile.value
 
       val log = state.value.log
 
-      def loadFile(file: File): String = {
-        val ciSource = Source.fromFile(file)
-        try {
-          ciSource.getLines().mkString("\n")
-        } finally {
-          ciSource.close()
-        }
-      }
-
       def reportMismatch(file: File, expected: String, actual: String): Unit = {
         log.error(s"Expected:\n$expected")
         log.error(s"Actual:\n${diff(expected, actual)}")
         sys.error(s"${file.getName} does not contain contents that would have been generated by sbt-github-actions; try running githubWorkflowGenerate")
       }
 
       def compare(file: File, expected: String): Unit = {
-        val actual = loadFile(file)
+        val actual = IO.read(file)
         if (expected != actual) {
           reportMismatch(file, expected, actual)
         }
       }
 
       compare(ciYml, expectedCiContents)
-      compare(cleanYml, expectedCleanContents)
+
+      if(includeClean)
```

Comment:
```suggestion
      if (includeClean)
```

You maniac…

Reply:
I rely on formatting tools to fix these for me :joy: